docs/eventlisteners.md

```diff
@@ -1,10 +1,75 @@
 # EventListener
 
-`EventListener`s connect `TriggerBinding`s to `TriggerTemplate`s and provide an
-addressable endpoint, which is where webhooks/events are directed. This is also
-where the service account is connected, which specifies what permissions the
-resources will be created (or at least attempted) with. The service account must
-have the following role bound.
+EventListener is a Kubernetes custom resource that allows users a declarative
+way to process incoming HTTP based events with JSON payloads. EventListeners
+expose an addressable "Sink" to which incoming events are directed. Users can
+declare [TriggerBindings](./triggerbindings.md) to extract fields from events,
+and apply them to [TriggerTemplates](./triggertemplates.md) in order to create
+Tekton resources. In addition, EventListeners allow lightweight event processing
+using [Event Interceptors](#event-Interceptors)
+
+- [Syntax](#syntax)
+  - [Triggers](#triggers)
+    - [Interceptors](#Interceptors)
+  - [ServiceAccountName](#serviceAccountName)
+- [Logging](#logging)
+- [Labels](#labels)
+- [Examples](#examples)
+
+## Syntax
+
+To define a configuration file for an `EventListener` resource, you can specify
+the following fields:
+
+- Required:
+  - [`apiVersion`][kubernetes-overview] - Specifies the API version, for example
+    `tekton.dev/v1alpha1`.
+  - [`kind`][kubernetes-overview] - Specifies the `EventListener` resource
+    object.
+  - [`metadata`][kubernetes-overview] - Specifies data to uniquely identify the
+    `EventListener` resource object, for example a `name`.
+  - [`spec`][kubernetes-overview] - Specifies the configuration information for
+    your EventListener resource object. In order for an EventListener to do
+    anything, the spec must include:
+    - [`triggers`](#triggers) - Specifies a list of Triggers to run
+    - [`serviceAccountName`](#serviceAccountName) - Specifies the serviceAccount
+      that the EventListener uses to create resources
+- Optional:
+  - [`serviceType`](#serviceType) - Specifies what type of service the sink pod
+    is exposed as
+
+[kubernetes-overview]:
+  https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields
+
+### Triggers
+
+The `triggers` field is required. Each EventListener can consist of one or more
+`triggers`. A Trigger consists of:
+
+- `name` - (Optional) a valid
+  [Kubernetes name](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)
+- [`interceptors`](#event-interceptors) - (Optional) list of interceptors to use
```

Comment:
`#event-Interceptors` link is broken now that the header is `Interceptors`

Reply:
done